.github/workflows/ros2foxy.yml

```diff
@@ -54,3 +54,36 @@ jobs:
         gazebo-version: ${{ matrix.gazebo }}
         workspace-dir: simulation_ws
         colcon-bundle-retries: 3
+    - name: Configure AWS Credentials
+      uses: aws-actions/configure-aws-credentials@v1.5.4
+      with:
+        aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID_ROS2 }}
+        aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY_ROS2 }}
+        aws-region: ${{ secrets.AWS_REGION }}
+      if: ${{ github.event_name == 'schedule' && contains(steps.robot_ws_build.outcome, 'success') && contains(steps.simulation_ws_build.outcome, 'success') }}
+    - id: upload_bundle
+      name: Upload bundle to S3
+      uses: aws-robotics/aws-robomaker-github-actions/s3-cp-action@2.4.2
```

Comment:
nit - do we want to keep the tag consistent within a given workflow?

Reply:
it's always good to use the latest -- will update